airsonic-main/src/test/java/org/airsonic/player/service/MediaScannerServiceTestCase.java

```diff
@@ -109,6 +109,7 @@ public Statement apply(Statement base, Description description) {
      */
     @Test
     public void testScanLibrary() {
+        musicFolderDao.getAllMusicFolders().forEach(musicFolder -> musicFolderDao.deleteMusicFolder(musicFolder.getId()));
```

Comment:
Isn't this pointing to the `airsonic-main/src/test/resources/MEDIAS/Music` and `airsonic-main/src/test/resources/MEDIAS/Music2`folders? I don't think we should be deleting these..

Reply:
By default, when server starts, database is filled with a default music folder named /var/music. 
If this folder exists and contains audio files it make tests fails. Thus it must be deleted.
Deleting all music folders is not a problem because test music folders are moked.

Reply:
What about using a random folder name somewhere in `/tmp`, to make sure that we won't remove existing music file from people that have a local airsonic instance running on their machine, with music in `/var/music`?

Reply:
Ah - I misunderstood this should be fine.